server/middlewares/csp.js

```diff
@@ -4,9 +4,13 @@ let trusted = [
   "'self'"
 ];
 
+const host = process.env.HOST || 'localhost';
+const port = process.env.PORT || process.env.SYNC_PORT || '3000';
+
 if (process.env.NODE_ENV !== 'production') {
   trusted = trusted.concat([
-    'ws://localhost:3000'
+    `http://${host}:${port}`,
```

Comment:
This line can be omitted as it is covered by `self`.

see: http://stackoverflow.com/questions/30280370/how-does-content-security-policy-work